pnpm-lock.yaml

```diff
@@ -41219,39 +41223,6 @@ packages:
       yargs-parser: 21.1.1
     dev: true
 
-  /ts-jest/29.1.1_gvmt7aj7nai5bnvsxmiksfugce:
-    resolution: {integrity: sha512-D6xjnnbP17cC85nliwGiL+tpoKN0StpgE0TeOjXQTU6MVCfsB4v7aW05CgQ/1OywGb0x/oy9hHFnN+sczTiRaA==}
```

Comment:
I wasn't expecting this to go away. (Didn't look deeper.)

Reply:
If I make the changes on a Mac, I get different lockfile changes. I updated it on Linux and pushed the results, so it should look cleaner now. Hope we can move to pnpm 8 soon.